src/utils/SiriusAdapter.cpp

```diff
@@ -131,49 +133,48 @@ class TOPPSiriusAdapter :
     registerInputFile_("in_featureinfo", "<file>", "", "FeatureXML input with feature and adduct information", false);
     setValidFormats_("in_featureinfo", ListUtils::create<String>("featureXML"));
 
-    registerOutputFile_("out_sirius", "<file>", "", "MzTab Output file for SiriusAdapter results", false);
+    registerOutputFile_("out_sirius", "<file>", "", "MzTab output file for SIRIUS results", false);
     setValidFormats_("out_sirius", ListUtils::create<String>("mzTab"));
 
     registerOutputFile_("out_fingerid","<file>", "", "MzTab output file for CSI:FingerID, if this parameter is given, SIRIUS will search for a molecular structure using CSI:FingerID after determining the sum formula", false);
     setValidFormats_("out_fingerid", ListUtils::create<String>("mzTab"));
 
+    registerOutputFile_("out_decoys","<file>", "", "Experimental output in project space.", false, true);
```

Comment:
Maybe remove - does not yet work in SIRIUSAdapter - another PR 
What would be needed here and why would someone need the decoys in this case?